CHANGELOG.md

```diff
@@ -4,8 +4,12 @@
 
 ### New features
 
+* `Autocomplete` introduces a new prop `compat-fallthrough`, which determines whether `class`, `style`, and `id` attributes are applied to the root `<div>` element or the underlying `<b-input>`.
```

Comment:
`Autocomplete` introduces a new prop `compat-fallthrough`, which determines whether the `class`, `style`, and `id` attributes are applied to the root `<div>` element or the underlying `<b-input>`.
  If `true`, they are applied to the root `<div>` element, which is compatible with Buefy for Vue 2.
  The default value can be controlled by the `defaultCompatFallthrough` config option (`true` by default).
  [#16](https://github.com/ntohq/buefy-next/issues/16)